### DISCUSSION TITLE
Stackoverflow profile

### DISCUSSION THREAD

**Discussion by eastriverlee** [community/community#48265] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I add Twitter link to my profile,
it gets a Twitter icon and a nice `@nickname` abbreviation. 
I think it's fair to say that being the most popular developer community,
Stackoverflow  deserves ***at least*** the same treatment in Github.

Currently it's lazily showing full link.
(i.e. https://stackoverflow.com/users/12922963/eastriver-lee)
Am I being unreasonable?

**Comment by tristan-f-r** [comment]
This makes sense - it is a popular dev forum, after all, and does deserve a spot despite it being unlike social media.

**Comment by vandan-revanur** [comment]
+1 Definitely makes sense to have abbreviated links support for Stackoverflow. And it would be awesome if all the other stack exchange sites are also included.

**Reply by tristan-f-r** [reply]
In that case, maybe a generic stack exchange logo can be supported. (Having specialized support for _all_ SE site logos doesn't make too much sense).

**Reply by vandan-revanur** [reply]
Yes a generic stack exchange logo should be good enough :)

**Comment by Skenvy** [comment]
Wondered why this wasn't already a thing, came looking to see if there was already a post asking for it; +1

**Comment by christianbraeunlich** [comment]
+1 Bump

**Comment by pjpscriv** [comment]
+1 for this. Stackoverflow has a custom icon for Github on their profile page - seems sporting to give them the same treatment.

![image](https://github.com/community/community/assets/16329923/c0a6fee0-5a74-4599-9752-15ba22a25245)

**Comment by pjpscriv** [comment]
While we wait for this to get prioritized (which I suspect may be some time 😅), I created a [userscript](https://gist.github.com/pjpscriv/6d29841107c93feeb042e730f29baea1) to do this. It's not perfect but it more or less does the job. 

It's pretty simple that it should be easy to copy and modify as you want :)

![image](https://github.com/user-attachments/assets/2a0d3a54-b5ba-4baf-9ac8-3bdbbf1d8070)

**Comment by pjpscriv** [comment]
I converted my userscript into a Chrome extension for easier use, check it out here https://chromewebstore.google.com/detail/github-social-icons/ledidlccnpfooopagmkgihgibfbnbbnp 😊

If username isn't included in the Stack Overflow url, it falls back to a `user:<USER_ID>` format

![image](https://github.com/user-attachments/assets/71f031ce-c43e-49cb-8b34-6836e3a55d56)

It also supports few different other types of links - if there are any others you want supported let me know!
